specifications/transfer-claim/transfer-claim.yml

```diff
@@ -803,7 +812,35 @@ components:
                     description: Percentage to finance of the claim total amount
                     minimum: 0
                     example: 90
-
+              claimStatus:
+                type: string
+                description: Status of account
+                enum:
+                  - active
+                  - parked
+                  - invoiced
+                  - closed
+                  - paid
+                  - waiting-for-payment
+                  - not-invoiced
+                  - late
```

Comment:
Actually, all the values you added in here are for all the types of accounts and not only for the orders.
you can check what is used for the orders from the code or asking to the Blue team ;)